### ISSUE TITLE
Page Configuration Translation

### ISSUE THREAD

**Issue by mrwhy-orig** [FluidTYPO3/flux#1137]
Hi,

I don't get any Page Configuration Translations.

I have 3 enabled languages German, English and Italian. But I just see the default language in the backend Page Configuration (German). 

Typo3 6.2.15
Flux 7.2.3

```
<f:section name="Configuration">
        <flux:form id="startseite" options="{icon: 'Icons/Page/Standard.gif'}" options="{translation: 'separate'}">
            <flux:field.select label="Navigation oder Info" name="isNav" items="{0:'Info', 1:'Navigation', 2:'NavNews'}">
            </flux:field.select>
            <flux:field.file label="Teaser Bild" name="image" allowed="jpg,png,gif" minItems="0" maxItems="1" size="1" />
            <flux:field.input label="Überschrift" name="headline" />
            <flux:field.text label="Teasertext" name="teasertext" rows="5" cols="30" enableRichText="TRUE" />
            <flux:field.input label="Link" name="teaserMehrLink">
                <flux:wizard.link activeTab="page" />
            </flux:field.input>
            <flux:field.select label="Buttonart" name='btnKind' items="{0:'Kein Link', 1:'Artikel Button', 2:'Back Button'}">

            </flux:field.select>
            <flux:field.input label="News Link" name="newsLink">
                <flux:wizard.link activeTab="page" />
            </flux:field.input>
            
        </flux:form>
        
        <flux:grid>
            <!-- Edit this grid to change the "backend layout" structure -->
            <flux:grid.row>
                <flux:grid.column colPos="0" name="Main" />
            </flux:grid.row>
        </flux:grid>
    </f:section>
```

Do I miss something or is it a bug?

Help is very welcome.

**Comment by NamelessCoder** [comment]
Hi @mrwhy-orig - see extension options of most recent version of fluidpages for a toggle that can add the  entire field to language overlay. Note that individual translation of form fields is no longer possible in TYPO3.

**Comment by mrwhy-orig** [comment]
Hi @NamlessCoder - what does most recent version mean? Is it available in a version which is compatible to the 6.2 LTS Version of Typo3?
Thx!

**Comment by NamelessCoder** [comment]
@mrwhy-orig This one would not be available in the 6.2-compatible versions (but you can implement the solution yourself by extending the SQL schema and page language overlay registry, you can use the exact same solution Fluidpages now ships with).
